README.md

```diff
@@ -48,8 +51,9 @@ Using Ruby, remove Belle from the town residents, and
 add her to the list of guests in the castle.
 
 Write your code here:
-```ruby
-# code here
+```
+residents.delete("Belle")
```

Comment:
`town[:residents]`, and try saving this to a variable. http://ruby-doc.org/core-2.2.0/Array.html#method-i-delete